modules/cleanup-a-backup-oadp-rosa-sts.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * rosa_backing_up_and_restoring_applications/backing-up-applications.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cleanup-a-backup-oadp-rosa-sts_{context}"]
+= Cleaning up after a backup with OADP and ROSA STS
```

Comment:
Title and description is misleading. This section contains information on how can we cleanup backup as well as the OADP operator and related CR's from the cluster.

Reply:
@PrasadJoshi12 

Should it be as follows:

Title - Cleaning a cluster after a backup with OADP and ROSA STS

Description - After you complete a backup with {oadp-first} and {product-rosa} (ROSA) {sts-full} (STS), you need to remove from the cluster the data of the backup, OADP Operator and other related custom resources (CRs).

Reply:
Imo Description should be something like this. 
If you want to uninstall the OADP operator along with the backups and S3 bucket, follow below instructions.

Reply:
@PrasadJoshi12 
Thank you!